macros/anaUltraThreshold.py

```diff
@@ -178,18 +283,42 @@
         pass
     pass
 outF.Close()
-txt = open(filename+"/vfatConfig.txt", 'w')
+txt_vfat = open(filename+"/vfatConfig.txt", 'w')
 
 print "trimRange:"
 print trimRange
 print "vt1:"
 print vt1
 
-txt.write("vfatN/I:vt1/I:vt1new/I:trimRange/I\n")
-for i in range(0,24):
-    txt.write('%i\t%i\t%i\t%i\n'%(i, vt1[i], vt1new[i], trimRange[i]))
+txt_vfat.write("vfatN/I:vt1/I:trimRange/I\n")
```

Comment:
vt1new was in the central repo since it was being used at p5. If not this algorithm for vt1 should be changed to the much better algorithm that starts at the maximum bin and increases until I finds a the first zero bin.